dessia_common/workflow/core.py

```diff
@@ -1941,6 +1943,119 @@ def _method_jsonschemas(self):
         warnings.warn("method_jsonschema method is deprecated. Use method_schema instead", DeprecationWarning)
         return self.method_schemas
 
+    @staticmethod
+    def generate_object_input_string(objects):
```

Comment:
These two methods (`generate_object_input_string` & `generate_file_input_string`) could still be functions detached from Workflow Class.

Now that you refactored them, they are really alike, and you could pass signature as an argument and simplify your calls in `process_sequence` methods

Reply:
done !